app/src/main/java/com/example/triptracker/view/map/AddSpot.kt

```diff
@@ -77,6 +82,15 @@ import com.example.triptracker.view.theme.md_theme_light_secondary
 import com.example.triptracker.view.theme.md_theme_orange
 import com.example.triptracker.viewmodel.RecordViewModel
 import com.google.android.gms.maps.model.LatLng
+import java.io.File
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+
+enum class AddSpotStatus {
```

Comment:
Maybe we could add a small comment for this enum class

Reply:
Done !